src/main/xml-parsers.js

```diff
@@ -248,15 +261,20 @@ export function parseListObjects(xml) {
   }
   var nextMarker
   var xmlobj = parseXml(xml)
-  if (xmlobj.IsTruncated && xmlobj.IsTruncated[0] === 'true') result.isTruncated = true
+
+  if (!xmlobj.ListBucketResult) {
+    throw new errors.InvalidXMLError('Missing tag: "ListBucketResult"')
+  }
+  xmlobj = xmlobj.ListBucketResult
+  if (xmlobj.IsTruncated && xmlobj.IsTruncated === 'true') xmlobj.isTruncated = true
   if (xmlobj.Contents) {
     xmlobj.Contents.forEach(content => {
```

Comment:
`xmlobj.Contents` will not be an array when the bucket has only one object. It throws the following errror

```
Uncaught TypeError: xmlobj.Contents.forEach is not a function
```

We need to add `Array.isArray` check here as well

Reply:
Sure. 
Will do.

Reply:
Done.